### DISCUSSION TITLE
Livewire 3 issue with Cloudflare

### DISCUSSION THREAD

**Discussion by uevandalla** [livewire/livewire#6704] [1. Help] [answered]
We've upgraded to livewire 3 and filament 3, we don't get any errors on our local development environments but when we deploy out to our staging environment which utilizes cloudflare we get the following errors:
![image](https://github.com/livewire/livewire/assets/137208695/bd7b8139-3b0b-44e1-835a-981795c6c4b4)

Mainly Uncaught Snapshot missing on Livewire component with id is the error we are concerned about, once this happens we can no longer do anything on the page as far as saving etc. we can refresh the page but then the same thing happens again.  I'm not sure how many others out there have already upgraded to livewire 3 for their staging or prod environments but wanted to see if we could get some help or insight into this issue.  When we shut off cloudflare the issue does go away but this is obviously not a suitable solution.  

Things we have tried:
We've disabled our cache in cloudflare, - issue remains
Disabled Cloudflare - errors go away but not acceptable solution
Added wild card in laravel middleware to accept all proxies - issue remains
Updated to latest livewire and filament versions - issue remains

**Comment by swaneswan** [comment]
I am also having a similar issue and have been unable to find a solution.

**Comment by ue-shuston** [comment]
This console error appears to be the same as the one reported in #6698.

**Comment by uevandalla** [comment]
we've narrowed the problem down to an issue using a subdomain with cloudflare.  using a regular domain seems to work just fine.  I'm not sure if this helps at all but wanted to add an update.

**Comment by RikoDEV** [comment, marked as answer]
I had a similar problem when using https://github.com/abordage/laravel-html-min/. After disabling the `remove_whitespace_between_tags` option, the problem did not occur. It's possible that cloudflare is minifying the final HTML, so livewire can't diff elements properly.

Try disabling all **Auto Minify** options in the cloudflare panel.

**Reply by uevandalla** [reply]
Sorry for the delay in marking the answer we wanted to be sure that this is truly what fixed it and it was.  Thank you @RikoDEV and @joshhanley

**Comment by joshhanley** [comment]
Yep, @RikoDEV is correct here, Cloudflare's auto minify will remove HTML comments - which Livewire relies upon - so disabling this should fix that.
